packages/api/tests/endpoints/async-operations/test-create-async-operations.js

```diff
@@ -0,0 +1,480 @@
+  await cleanupTestIndex(t.context);
+  await recursivelyDeleteS3Bucket(process.env.system_bucket);
+  await destroyLocalTestDb({
+    knex: t.context.testKnex,
+    knexAdmin: t.context.testKnexAdmin,
+    testDbName,
+  });
+});
+
+test('POST without an Authorization header returns an Authorization Missing response', async (t) => {
+  const asyncOperation1 = {
+    id: uuidv4(),
+    status: 'RUNNING',
+    taskArn: randomString(),
+    description: 'Some async run',
+    operationType: 'Bulk Granules',
+    output: JSON.stringify({ age: 59 }),
+  };
+
+  const response = await request(app)
+    .post('/asyncOperations')
+    .send(asyncOperation1)
+    .set('Accept', 'application/json')
+    .expect(401);
+
+  assertions.isAuthorizationMissingResponse(t, response);
+  await t.throwsAsync(
+    () => t.context.asyncOperationModel.get({ id: asyncOperation1.id }),
+    { instanceOf: RecordDoesNotExist }
+  );
+});
+
+test('POST with an invalid access token returns an unauthorized response', async (t) => {
+  const asyncOperation = {
+    id: uuidv4(),
+    status: 'RUNNING',
+    taskArn: randomString(),
+    description: 'Some async run',
+    operationType: 'Bulk Granules',
+    output: JSON.stringify({ age: 59 }),
+  };
+
+  const response = await request(app)
+    .post('/asyncOperations')
+    .send(asyncOperation)
+    .set('Accept', 'application/json')
+    .set('Authorization', 'Bearer ThisIsAnInvalidAuthorizationToken')
+    .expect(401);
+
+  assertions.isInvalidAccessTokenResponse(t, response);
+  await t.throwsAsync(
+    () => t.context.asyncOperationModel.get({ id: asyncOperation.id }),
+    { instanceOf: RecordDoesNotExist }
+  );
+});
+
+test('POST creates a new async operation in all data stores', async (t) => {
+  const { asyncOperationPgModel, jwtAuthToken } = t.context;
+  const asyncOperation = {
+    id: uuidv4(),
+    status: 'RUNNING',
+    taskArn: randomString(),
+    description: 'Some async run',
+    operationType: 'Bulk Granules',
+    output: JSON.stringify({ age: 59 }),
+  };
+
+  const pgAsyncOperation = await translateApiAsyncOperationToPostgresAsyncOperation(asyncOperation);
+
+  const omitList = ['created_at', 'updated_at', 'cumulus_id', 'output'];
+
+  const response = await request(app)
+    .post('/asyncOperations')
+    .send(asyncOperation)
+    .set('Accept', 'application/json')
+    .set('Authorization', `Bearer ${jwtAuthToken}`)
+    .expect(200);
+
+  const { message, record } = response.body;
+  const pgRecord = await asyncOperationPgModel.search(
+    t.context.testKnex,
+    { id: asyncOperation.id }
+  );
+
+  t.is(message, 'Record saved');
+  t.is(record.id, asyncOperation.id);
+  t.is(pgRecord.length, 1);
+
+  const [asyncOperationPgRecord] = pgRecord;
+
+  t.deepEqual(
+    omit(asyncOperationPgRecord, omitList),
+    omit(pgAsyncOperation, omitList)
+  );
+  t.deepEqual(asyncOperationPgRecord.output, JSON.parse(pgAsyncOperation.output));
```

Comment:
Hmm this is confusing to me. I believe the `output` comes out of PG as an object , not a JSON string, which would agree with the `PostgresAsyncOperation` type. So doing a `JSON.parse` on something that is already an object 

I might need to delve a little more deeply here